modules/mantisBidAdapter.js

```diff
@@ -0,0 +1,221 @@
+import {registerBidder} from 'src/adapters/bidderFactory';
+
+function inIframe() {
+  try {
+    return window.self !== window.top && !window.mantis_link;
+  } catch (e) {
+    return true;
+  }
+}
+
+function isDesktop(ignoreTouch) {
+  var supportsTouch = !ignoreTouch && ('ontouchstart' in window || navigator.msMaxTouchPoints);
+  if (inIframe()) {
+    return !supportsTouch;
+  }
+  var width = window.innerWidth || window.document.documentElement.clientWidth || window.document.body.clientWidth;
+  return !supportsTouch && (!width || width >= (window.mantis_breakpoint || 768));
+}
+
+function storeUuid(uuid, sync) {
+  if (window.mantis_uuid) {
+    return;
+  }
+  window.mantis_uuid = uuid;
+  if (window.localStorage) {
+    try {
+      window.localStorage.setItem('mantis:uuid', uuid);
+    } catch (ex) {
+    }
+  }
+  if (!sync) {
+    postMessage('uuid', uuid);
```

Comment:
what's the purpose of the postMessage?

Reply:
@mkendall07 this was common code for frame syncing for our non-prebid code, build now removes it. thanks